packages/ckb/package.json

```diff
@@ -18,18 +18,20 @@
   ],
   "types": "./lib/index.d.ts",
   "dependencies": {
-    "@ckb-lumos/base": "0.21.1",
-    "@ckb-lumos/codec": "0.22.0-next.5",
+    "@ckb-lumos/base": "^0.21.1",
+    "@ckb-lumos/codec": "^0.22.1",
+    "@spore-sdk/core": "^0.2.0-beta.6",
     "@nervosnetwork/ckb-sdk-core": "^0.109.1",
     "@nervosnetwork/ckb-sdk-utils": "^0.109.1",
     "@nervosnetwork/ckb-types": "^0.109.1",
     "@rgbpp-sdk/service": "workspace:^",
+    "@exact-realty/multipart-parser": "^1.0.13",
```

Comment:
This package is not used?

Reply:
The `lumos`  SDK and `spore`  SDK need the `@exact-realty/multipart-parser`, without it, they will occur error.

see: Nervape Tech telegram group context